brainstormforce/ultimate-addons-for-gutenberg
# Fix: Section - Border radius issue on frontend.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/4d53bd0d-bb85-4209-9a43-7e3c0d0e4077?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- Output - https://share.getcloudapp.com/04uLZ86N

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create a Section with border and bg color in master.
- Switch to next-release and refresh the frontend you will see an issue.
- Switch to above branch and check if issue is fix or not.

### Checklist:
